### ISSUE TITLE
Send PLAY or PAUSE

### ISSUE THREAD

**Issue by Isaksson** [QNimbus/node-red-contrib-openhab-v2#7]
I updated this node today and now I cant control my Sonos speaker.
If i try to send payload PLAY or PAUSE then i receive error message 400 Bad request, is this a bug in this new version?
I noticed that you did not include the command PLAY or PAUSE when selecting OpenHAB as Payload.

**Comment by QNimbus** [comment]
I have not yet added these specific commands to the OpenHAB command type list - I will do this when I get to it. Could you give me more information on what node type(s) you are using and how you have it configured when trying to send 'PLAY' or 'PAUSE' payload? Maybe a screenshot of your flow and node configuration - I will try to figure out what is happening and see if I can help or resolve the issue.

**Comment by QNimbus** [comment]
I had not yet tried to make a flow for my Sonos players - but to test it I made a quick 'n dirty flow to PLAY and PAUSE my kitchen player as seen in the screenshots. This seems to work as it should.

![nodes](https://user-images.githubusercontent.com/3206309/46723671-a5868b00-cc78-11e8-9058-ebd8150fb745.PNG)

![config](https://user-images.githubusercontent.com/3206309/46723694-b0d9b680-cc78-11e8-9ab9-19b171a1478c.PNG)

**Comment by Isaksson** [comment]
I have now made some more test and my first assumption was wrong, the problem is not to send Play Or Pause or something like that, the problem was that i updated from an older version and already had a lot of flows going, I had to modify every instance of this node, select the item and Topic and Payload.
The problem is that it was not enough to enter msg.payload as Payload I have to first select OpenHab and then back to msg. to get things up and running.

I guess this has something to do with the new version and my "old" nodes if I dont change and change back then the parameter is not saved.

If I add a new node then I do not have to change and then back to msg.

So the issue title is not accurate :) it should be something like upgrade from older version breaks my nodes, 
solution: select OpenHAB as payload and then back to msg. (if using payload as command)
